src/pyrokinetics/gk_code/gs2.py

```diff
@@ -62,6 +64,15 @@ class GKInputGS2(GKInput, FileReader, file_type="GS2", reads=GKInput):
         "beta_prime": ["theta_grid_eik_knobs", "beta_prime_input"],
     }
 
+    pyro_gs2_mxh = {
+        **pyro_gs2_miller,
+        "c0": ["theta_grid_parameters", "c0_mxh"],
```

Comment:
GS2 no longer has a separate c0 input so we should remove this and similar later variables.